inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/DefaultMessageSender.java

```diff
@@ -218,6 +226,9 @@ public SendResult sendMessage(byte[] body, String groupId, String streamId, long
      */
     public SendResult sendMessage(byte[] body, String groupId, String streamId, long dt, String msgUUID,
             long timeout, TimeUnit timeUnit, boolean isProxySend) {
+        if (enableDataTruncation) {
+            body = truncateData(body);
```

Comment:
If a truncation is made, how can we ensure that the transmitted data is the user-reported data?

Reply:
Sorry, do you mean to add data verification function?

Reply:
The data content reported by the user cannot be modified